.github/workflows/build.yaml

```diff
@@ -65,6 +65,9 @@ env:
   # URL to get source code for building the image
   IMAGE_SRC: https://github.com/radius-project/radius
 
+  # ACR url for bicep file uploads
+  ACR: 'biceptypes.azurecr.io'
```

Comment:
Can you come up with a more descriptive name for this? It looks like my comment disappeared, but its still the same.

Reply:
Ah, sorry I didnt see that comment. I updated to BICEP_TYPES_ACR, but I'm open to suggestions if something else seems better

Reply:
I'd suggest something like `BICEP_TYPES_REGISTRY`. It's more neutral.

Reply:
Updated